types/errors/callbacks.go

```diff
@@ -0,0 +1,20 @@
+package errors
+
+import (
+	"log"
+	"os"
+)
+
+var logger = log.New(os.Stderr, "defer_callback: ", log.LstdFlags|log.LUTC|log.Llongfile)
```

Comment:
If this is used in `cobra`'s Commands, this would get around `cobra` stdout and stderr redirection. Again, it's harmful.

Reply:
Why it's harmful? We can iterate over it later on and use other logger. I don't have a time to refactore many parts of the SDK now.